include/overlay016/struct_ov16_02264408.h

```diff
@@ -25,7 +25,7 @@ typedef struct {
     u32 unk_4C[4];
     int unk_50;
     UnkStruct_ov16_02264408_sub1 unk_54;
-    UnkStruct_0202CC84 * unk_6C;
+    ChatotCryData * unk_6C;
```

Comment:
```suggestion
    ChatotCryData *chatotCry;
```

(and elsewhere in the PR)